test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -50,7 +50,7 @@ class AndroidArgs(
     val appApk = cli.app ?: androidGcloud.app
     val testApk = cli.test ?: androidGcloud.test
     val autoGoogleLogin = androidGcloud.autoGoogleLogin
-    val useOrchestrator = androidGcloud.useOrchestrator
+    val useOrchestrator = cli.useOrchestrator ?: cli.noUseOrchestrator?.not() ?: androidGcloud.useOrchestrator
```

Comment:
`not()` because if the flag is on, useOrchestrator needs to be `false`

Reply:
let's add this as a comment, the double negative is confusing

`no.not()` 😂